src/screening/ScreeningForm.jsx

```diff
@@ -16,7 +16,8 @@ class ScreeningForm extends React.Component {
     const formSubmittedActionCreators = [
       () => props.patient && props.patient.uuid && visitActions.fetchPatientActiveVisit(props.patient.uuid,
         this.props.sessionLocation ? this.props.sessionLocation.uuid : null, ACTIVE_VISITS_REP),
-      () => props.patient && props.patient.uuid && patientActions.getIC3PatientScreeningData(props.patient)
+      () => props.patient && props.patient.uuid && patientActions.getIC3PatientScreeningData(props.patient),
+      () => props.patient && props.patient.uuid && patientActions.getIC3PatientNutritionHistory(props.patient),
```

Comment:
Same as my comment from above... can we just do this when the ClinicianSummary or NutritionSummary pages load? Do we need to do it every time any ScreeningForm is loaded?

Reply:
@mogoodrich so this is for getting the  updated ic3 nutrition history after saving the form, I was hoping that the nutritionSummary and clinicianSummary would both use data from one part of the store

Reply:
I dropped a reference below